.cursor/rules/accessibility-requirements.mdc

```diff
@@ -0,0 +1,271 @@
+---
+alwaysApply: true
+---
+
+# Accessibility: Provide Meaningful Content for Screen Readers
+
+## Principle
+
+Ensure all UI elements provide meaningful content for screen readers and assistive technologies. Use VisuallyHidden for accessibility instead of empty strings or missing labels.
+
+## Guidelines
+
+### Do This ✅
+
+```typescript
+// Good: Provide meaningful screen reader content
+import { VisuallyHidden } from '@/components/Common'
+
+const DataTable = ({ data }) => (
+  <table>
+    <thead>
+      <tr>
+        <th>
+          Employee Name
+          <VisuallyHidden>Sort by employee name</VisuallyHidden>
+        </th>
+        <th>
+          Pay Amount
+          <VisuallyHidden>Sort by pay amount</VisuallyHidden>
+        </th>
+        <th>
+          <VisuallyHidden>Actions for employee</VisuallyHidden>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      {data.map(employee => (
+        <tr key={employee.id}>
+          <td>{employee.name}</td>
+          <td>{employee.amount}</td>
+          <td>
+            <Button aria-label={`Edit ${employee.name}`}>
+              <Icon name="edit" />
+              <VisuallyHidden>Edit employee</VisuallyHidden>
+            </Button>
+          </td>
+        </tr>
+      ))}
+    </tbody>
+  </table>
+)
+
+// Good: Meaningful button labels
+const PayrollActions = ({ payroll }) => (
+  <div>
+    <Button
+      aria-label={`Submit payroll for ${payroll.payPeriod}`}
+      onClick={() => submitPayroll(payroll.id)}
+    >
+      <Icon name="submit" />
+      <VisuallyHidden>Submit payroll</VisuallyHidden>
+    </Button>
+
+    <Button
+      aria-label={`Cancel payroll for ${payroll.payPeriod}`}
+      onClick={() => cancelPayroll(payroll.id)}
+    >
+      <Icon name="cancel" />
+      <VisuallyHidden>Cancel payroll</VisuallyHidden>
+    </Button>
+  </div>
+)
+```
```

Comment:
Same on these ones, these should be updated to be `IconButton` instances with aria-label set